src/Dashboard/Data/BlobPath.cs

```diff
@@ -47,7 +47,7 @@ public static BlobPath Parse(string value)
 
             if (!TryParse(value, out path))
             {
-                throw new FormatException("Blob identifiers must be in the format 'container/blob'.");
+                throw new FormatException($"Blob identifiers must be in the format 'container/blob', but given '{value}'.");
```

Comment:
Recommend rewording to "Invalid blob path specified : '{value}'. Blob identifiers must be in the format 'container/blob'."